# 1. Ejercicios

#### 1 Ejercicio

Python funciona como una calculadora avanzada. Escribe en Python las siguientes expresiones:
1. $2 * 3 / 2$
1. $\frac{40}{3}$
1. $(\frac{4}{2})^5$
1. $(-2)^2$
1. $-(3)^3$
2. $\frac{4+6}{2+3}$
2. $2 + (3 * (\frac{6}{2}))$
2. $5^{5}$



In [10]:
# Sol 1:
print(2*32)
print(40/3)
print((4/2)**5)
print((-2)**2)
print(-3**3)
print((4+6)/(2+3))
print(2+(3*(6/2)))
print(5**5)

64
13.333333333333334
32.0
4
-27
2.0
11.0
3125


#### 2 Ejercicio

Para calcular el importe de IVA de $ 145$ Euros es necesario multiplicar $145$ por $0.21$.
Escribe en Python la expresión que calcula el IVA de $145$ Euros.

In [1]:
# La expresión resulta de 145x21/100
print(145*0.21)

30.45


Escribe en Python la expresión que calcula el IVA de $258$ Euros.

In [2]:
print(258*0.21)

54.18


Escribe una función en Python llamada __calcula_Iva__ que calcule el importe de IVA de una cierta cantidad $x$.

In [3]:
def calcula_Iva(importe):
    return importe * 0.21

In [5]:
calcula_Iva(145)

30.45

In [6]:
calcula_Iva(258)

54.18

Ejecuta la función __calcula_Iva__ para las cantidades $145$ y $258$. Comprueba que los resultados son $30.45$ y $54.18$

#### 3 Ejercicio

Evalúa el polinomio $ x^4 + x^3 + \frac{1}{2} x^2 - x $
en $ x = 10 $. El resultado
ha de ser  un número real.

In [8]:
x = 10
calculo = x**4+x**3+1/2*x**2-x
print("El resultado es",calculo)
# El resultado es 11040.0

El resultado es 11040.0


Evalúa  el polinomio anterior para $x = 7$.

In [9]:
x = 7
calculo = x**4+x**3+1/2*x**2-x
print("El resultado es",calculo)

El resultado es 2761.5


#### 4 Ejercicio

Define una función llamada __raiz_cubica__ que devuelva el valor de $\sqrt[3]{x}$. Recuerda que  $\sqrt[3]{x}$ = ${x}^{\frac{1}{3}}$


In [44]:
x = 5
def raiz_cubica(x):
    return x**1/3
a = raiz_cubica(x)
print ('El resultado de la raiz cubica de %d es %.1f0' % (x, a))

El resultado de la raiz cubica de 5 es 1.70


Escribe la expresión Python que construya la siguiente cadena:

$$"El \ resultado \ de  \  la  \  raíz  \  cúbica  \   de  \ 5 \  es:  1.70"$$

Utiliza el operador de formato __%f__.

--------------------

<a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/"><img alt="Licencia Creative Commons" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br />